Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -685,15 +704,25 @@ private void ApplyTargetTransform(MixedRealityTransform targetTransform)
             }
             else
             {
-                rigidBody.velocity = ((1f - Mathf.Pow(moveLerpTime, Time.deltaTime)) / Time.deltaTime) * (targetTransform.Position - HostTransform.position);
-
-                var relativeRotation = targetTransform.Rotation * Quaternion.Inverse(HostTransform.rotation);
-                relativeRotation.ToAngleAxis(out float angle, out Vector3 axis);
-                if (angle > 180f)
-                    angle -= 360f;
-                if (axis.IsValidVector())
+                if (!useForcesForNearManipulation && IsNearManipulation())
+                {
+                    // If not using forces for near manipulation and this is a near manipulation, apply immediate updates
+                    rigidBody.MovePosition(targetTransform.Position);
+                    rigidBody.MoveRotation(targetTransform.Rotation);
+                }
+                else
                 {
-                    rigidBody.angularVelocity = ((1f - Mathf.Pow(rotateLerpTime, Time.deltaTime)) / Time.deltaTime) * (axis.normalized * angle * Mathf.Deg2Rad);
+                    // Far manipulation is always applied using forces
```

Comment:
Nit: The comment makes it seem like this is specifically for far manip, but it's also near as well.

Reply:
Happy to fix this Nit. Are there other issues on this one? I was hoping some folks like perhaps @julenka could help test this proposed change. Specifically I've noticed that the holograms don't always appear to attach to my hand where I expect them to. Especially if the holograms are far below my head (like on the ground), I will see a significant offset between the hologram and my hand. But I'm pretty sure this was actually there before and not due to my proposed change. I just want to make sure that's the case, and if so I can open a separate issue on that.